client/state/selectors/get-post-revision-changes.js

```diff
@@ -4,22 +4,53 @@
  * External dependencies
  */
 
-import { findIndex } from 'lodash';
+import { findIndex, get, isUndefined, map, reduce, omitBy } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import createSelector from 'lib/create-selector';
+import { countDiffWords, diffWords } from 'lib/text-utils';
 import getPostRevisions from 'state/selectors/get-post-revisions';
 
+const MAX_DIFF_WORDS = 10000;
```

Comment:
Might make sense to do some more in-depth testing to figure out at which point it fails us. Given that technically this can/will run on a mobile device, the number might be lower than we think 😕

Reply:
I def agree we need to do some testing.

To clarify, that's not currently limiting at 10000 words, but combined string length between the revisions' content & title.